web/src/user_pill.js

```diff
@@ -85,6 +86,7 @@ export function append_person(opts) {
 
     pill_widget.appendValidatedData(pill_data);
     pill_widget.clear_text();
+    compose_recipient.update_placeholder_text();
```

Comment:
I'm simply wondering, if the `update_placeholder_text` wasn't triggered when adding a person, what was responsible for updating the placeholder previously? Shouldn't we address that part which used to update the placeholder?